frontend/src/scenes/surveys/constants.tsx

```diff
@@ -43,6 +43,9 @@ export const defaultSurveyAppearance = {
     displayThankYouMessage: true,
     thankYouMessageHeader: 'Thank you for your feedback!',
     position: 'right',
+    widgetType: 'tab' as const,
+    widgetLabel: 'Feedback',
+    widgetColor: 'black',
```

Comment:
do those values cause any issue if the chosen survey type is `popup` or `api`? it should not be part of the json example, or part of the created survey if the type isn't `feedback button`

Reply:
they don't, but I've sanitized them to remove if survey type isn't widget now